coco_tools/merge.py

```diff
@@ -0,0 +1,2 @@
+def merge():
+    print("Unimplemented!")
```

Comment:
Although this is not an abstract base class, could also use the build-in `NotImplementedError`. Just a comment, no need to change